public_html/wp-content/plugins/wordcamp-payments/includes/reimbursement-request.php

```diff
@@ -257,7 +257,12 @@ function enqueue_assets() {
  * @return bool
  */
 function user_can_edit_request( $post ) {
-	$editable_status = in_array( $post->post_status, array( 'auto-draft', 'draft', 'wcb-incomplete' ), true );
+	$editable_status = false;
+
+	if ( $post instanceof WP_Post ) {
+		$editable_status = in_array( $post->post_status, array( 'auto-draft', 'draft', 'wcb-incomplete' ), true );
+	}
+
```

Comment:
Was this added in by mistake?

Reply:
Bah, that was to hide a php error that was showing when you click the History row link for a feedback comment. But since it's otherwise unrelated, I should probably just commit that directly to production instead.

Reply:
1124076b1f350e24fbb247cc59b9a521dfa15225